<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JAL_MainWindow</class>
 <widget class="QMainWindow" name="JAL_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1482</width>
    <height>583</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>jal</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="MainTabs">
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="HoldingsTab">
       <attribute name="title">
        <string>Holdings</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="HoldingsParamsFrame">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>2</number>
           </property>
           <property name="topMargin">
            <number>2</number>
           </property>
           <property name="rightMargin">
            <number>2</number>
           </property>
           <property name="bottomMargin">
            <number>2</number>
           </property>
           <item>
            <widget class="QDateEdit" name="HoldingsDate">
             <property name="dateTime">
              <datetime>
               <hour>21</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2020</year>
               <month>11</month>
               <day>29</day>
              </datetime>
             </property>
             <property name="displayFormat">
              <string>dd/MM/yyyy</string>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
             <property name="timeSpec">
              <enum>Qt::UTC</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="HoldingsCurrencyLbl">
             <property name="text">
              <string>Common currency:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="CurrencyComboBox" name="HoldingsCurrencyCombo"/>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>1411</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTreeView" name="HoldingsTableView">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="animated">
           <bool>true</bool>
          </property>
          <property name="allColumnsShowFocus">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ReportsTab">
       <attribute name="title">
        <string>Reports</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="ReportParamsFrame">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <property name="leftMargin">
            <number>2</number>
           </property>
           <property name="topMargin">
            <number>2</number>
           </property>
           <property name="rightMargin">
            <number>2</number>
           </property>
           <property name="bottomMargin">
            <number>2</number>
           </property>
           <property name="spacing">
            <number>6</number>
           </property>
           <item row="0" column="10">
            <spacer name="ReportFrameSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="2">
            <widget class="AccountButton" name="ReportAccountBtn"/>
           </item>
           <item row="0" column="9">
            <widget class="CategorySelector" name="ReportCategoryEdit" native="true"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="ReportTypeLbl">
             <property name="text">
              <string>Report:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="5">
            <widget class="QDateEdit" name="ReportToDate">
             <property name="dateTime">
              <datetime>
               <hour>21</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2020</year>
               <month>11</month>
               <day>29</day>
              </datetime>
             </property>
             <property name="displayFormat">
              <string>dd/MM/yyyy</string>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
             <property name="timeSpec">
              <enum>Qt::UTC</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="ReportAccountLbl">
             <property name="text">
              <string>Account:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3" rowspan="2">
            <widget class="Line" name="line_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QLabel" name="ReportToLbl">
             <property name="text">
              <string>To:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QLabel" name="ReportFromLbl">
             <property name="text">
              <string>From:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="6">
            <widget class="QCheckBox" name="ReportGroupCheck">
             <property name="text">
              <string>Group dates</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QComboBox" name="ReportTypeCombo">
             <item>
              <property name="text">
               <string>Income / Spending</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Profit / Loss</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Deals</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>By Category</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="11">
            <widget class="QPushButton" name="SaveReportBtn">
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
           <item row="0" column="11">
            <widget class="QPushButton" name="RunReportBtn">
             <property name="text">
              <string>Run</string>
             </property>
            </widget>
           </item>
           <item row="0" column="8">
            <widget class="QLabel" name="ReportCategoryLbl">
             <property name="text">
              <string>Category:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="6">
            <widget class="QComboBox" name="ReportRangeCombo">
             <item>
              <property name="text">
               <string>Custom range</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Quarter to date</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Year to date</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>This year</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Previous year</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QDateEdit" name="ReportFromDate">
             <property name="dateTime">
              <datetime>
               <hour>21</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2020</year>
               <month>11</month>
               <day>29</day>
              </datetime>
             </property>
             <property name="displayFormat">
              <string>dd/MM/yyyy</string>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
             <property name="timeSpec">
              <enum>Qt::UTC</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="7" rowspan="2">
            <widget class="Line" name="line_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="ReportTableView">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="gridStyle">
           <enum>Qt::DotLine</enum>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>20</number>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>20</number>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QTreeView" name="ReportTreeView">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <attribute name="headerStretchLastSection">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="TabbedMdiArea" name="mdiArea" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="MainMenu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1482</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Data">
    <property name="title">
     <string>&amp;Data</string>
    </property>
    <widget class="QMenu" name="menuPredefined_data">
     <property name="title">
      <string>Predefined data</string>
     </property>
     <addaction name="actionAccountTypes"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="actionAccounts"/>
    <addaction name="actionAssets"/>
    <addaction name="actionPeers"/>
    <addaction name="actionCategories"/>
    <addaction name="actionTags"/>
    <addaction name="actionCountries"/>
    <addaction name="actionQuotes"/>
    <addaction name="menuPredefined_data"/>
    <addaction name="separator"/>
    <addaction name="actionBackup"/>
    <addaction name="actionRestore"/>
    <addaction name="separator"/>
    <addaction name="action_Re_build_Ledger"/>
   </widget>
   <widget class="QMenu" name="menu_Export">
    <property name="title">
     <string>&amp;Export</string>
    </property>
    <addaction name="PrepareTaxForms"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>L&amp;anguage</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuImport">
    <property name="title">
     <string>&amp;Import</string>
    </property>
    <widget class="QMenu" name="menuStatement">
     <property name="title">
      <string>&amp;Statement</string>
     </property>
    </widget>
    <addaction name="action_LoadQuotes"/>
    <addaction name="menuStatement"/>
    <addaction name="actionImportSlipRU"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Data"/>
   <addaction name="menuImport"/>
   <addaction name="menu_Export"/>
   <addaction name="menuLanguage"/>
  </widget>
  <widget class="QStatusBar" name="StatusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="action_Re_build_Ledger">
   <property name="text">
    <string>Re-build &amp;Ledger...</string>
   </property>
  </action>
  <action name="action_LoadQuotes">
   <property name="text">
    <string>&amp;Quotes...</string>
   </property>
  </action>
  <action name="actionImportStatement">
   <property name="text">
    <string>&amp;Broker statement...</string>
   </property>
  </action>
  <action name="actionAccountTypes">
   <property name="text">
    <string>Account &amp;Types</string>
   </property>
  </action>
  <action name="actionAccounts">
   <property name="text">
    <string>&amp;Accounts</string>
   </property>
  </action>
  <action name="actionAssets">
   <property name="text">
    <string>A&amp;ssets</string>
   </property>
  </action>
  <action name="actionPeers">
   <property name="text">
    <string>&amp;Peers</string>
   </property>
  </action>
  <action name="actionCategories">
   <property name="text">
    <string>&amp;Categories</string>
   </property>
  </action>
  <action name="actionBackup">
   <property name="text">
    <string>&amp;Backup...</string>
   </property>
  </action>
  <action name="actionRestore">
   <property name="text">
    <string>&amp;Restore...</string>
   </property>
  </action>
  <action name="PrepareTaxForms">
   <property name="text">
    <string>&amp;Tax report [RU]</string>
   </property>
  </action>
  <action name="MakeDealsReport">
   <property name="text">
    <string>&amp;Deals report</string>
   </property>
  </action>
  <action name="actionTags">
   <property name="text">
    <string>&amp;Tags</string>
   </property>
  </action>
  <action name="MakePLReport">
   <property name="text">
    <string>&amp;Profit/Loss report</string>
   </property>
  </action>
  <action name="MakeCategoriesReport">
   <property name="text">
    <string>&amp;Income/Spending report</string>
   </property>
  </action>
  <action name="actionImportSlipRU">
   <property name="text">
    <string>Slip [RU]...</string>
   </property>
  </action>
  <action name="actionCountries">
   <property name="text">
    <string>C&amp;ountries</string>
   </property>
  </action>
  <action name="actionQuotes">
   <property name="text">
    <string>&amp;Quotes</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AccountButton</class>
   <extends>QPushButton</extends>
   <header>jal/widgets/account_select.h</header>
  </customwidget>
  <customwidget>
   <class>CurrencyComboBox</class>
   <extends>QComboBox</extends>
   <header>jal/widgets/account_select.h</header>
  </customwidget>
  <customwidget>
   <class>CategorySelector</class>
   <extends>QWidget</extends>
   <header>jal/widgets/reference_selector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabbedMdiArea</class>
   <extends>QWidget</extends>
   <header>jal/widgets/tabbed_mdi_area.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
